packages/ceramic-cli/src/bin/ceramic.ts

```diff
@@ -25,7 +25,7 @@ program
     .option('--content <content>', 'New document content')
     .option('--only-genesis', 'Only create the genesis object. No anchor will be created')
     .option('--controllers <controllers>', 'Specify a comma-separated list of the controllers of the document. Controllers are the users that are allowed to publish updates to this document. Defaults to current user')
-    .option('--unique', 'Ensure document is unique regardless of content')
+    .option('--unique <unique>', 'Controls whether document is ensured to be unique regardless of content. Defaults to true')
```

Comment:
I wasn't sure how to test the behavior of the CLI.  I assume there's no automated tests and this will have to be done manually?  I guess that means it's time for me to figure out how to deploy a ceramic node locally and use the CLI with it.  Can you link me to the best starting point docs on that?

Reply:
Yep, you're right. We're still working on more detailed documentation for the CLI but you can start from [here](https://medium.com/ceramic/building-with-clay-getting-started-with-the-ceramic-network-3d7a8a1e01bf). The complete local environment should be completed as part of the story [395 ](https://github.com/ceramicnetwork/js-ceramic/issues/395).